src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -387,9 +388,15 @@ void MainWindow::build_menus()
     m_ui->menu_view->addAction(m_ui->python_console->toggleViewAction());
     m_ui->menu_view->addSeparator();
 
-    QAction* fullscreen_action = m_ui->menu_view->addAction("Fullscreen");
-    fullscreen_action->setShortcut(Qt::Key_F11);
-    connect(fullscreen_action, SIGNAL(triggered()), SLOT(slot_fullscreen()));
+    m_action_fullscreen = m_ui->menu_view->addAction("Fullscreen");
+    m_action_fullscreen->setCheckable(true);
+    m_action_fullscreen->setShortcut(Qt::Key_F11);
+    m_action_fullscreen->setObjectName("Fullscreen");
```

Comment:
Remember my previous comment about the name of the action? It turns out I had simply commented on the wrong line :) The argument to `setObjectName()` should be `action_fullscreen`. But I wonder if we really need a name for that action; I think not, in which case please remove the call altogether.

Reply:
Nope, we don't need it so I removed it. :smile: